api/src/main/resources/db/migration/1.0/V1.0.72__DDL-REPLACE_VIEW-REPORT_GRAD_SCHOOL__DISTRICT_YE_VW.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW REPORT_GRAD_SCHOOL_YE_VW AS
+SELECT
+    COUNT(GRADUATION_STUDENT_RECORD_ID) STUD_NUM,
+    -- Revert code back to school of record GRAD2-2758
+    --SUBSTR(DECODE(A1.SCHOOL_AT_GRADUATION, NULL, A1.SCHOOL_OF_RECORD, A1.SCHOOL_AT_GRADUATION), 0, 3) MINCODE
+    A1.SCHOOL_OF_RECORD MINCODE,
```

Comment:
can we get rid of mincode?
Do we need to update REPORT_GRAD_DISTRICT_YE_VW as well?

Reply:
Mincode is removed from REPORT_GRAD_SCHOOL_YE_VW.
REPORT_GRAD_DISTRICT_YE_VW has been dropped.